overlord/servicestate/quota_control_test.go

```diff
@@ -921,3 +923,97 @@ func (s *quotaControlSuite) TestCreateQuotaCreateQuotaConflict(c *C) {
 	_, err = servicestate.CreateQuota(st, "foo", "", []string{"test-snap2"}, 2*quantity.SizeGiB)
 	c.Assert(err, ErrorMatches, `quota group "foo" has "quota-control" change in progress`)
 }
+
+func (s *quotaControlSuite) TestMemoryCGroupDisabled(c *C) {
```

Comment:
we also need a variant of this test when the file doesn't exist or when it's in an invalid format we don't support to make sure that the functions return sensible error messages